metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/AsyncListener.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *  Copyright 2016 Netflix, Inc.
```

Comment:
You added the license but the year is wrong. You should change your intellij settings to auto add license to all new classes and you can have a placeholder I think it's like `${year}` or similar and it will auto sub it in every time you create a new class.